### PR TITLE
Fix URL in README plus upgraded 9 gems

### PR DESCRIPTION
**Pull request by jasnow**
* Fix URL (from .com to .org) in README file.
 * Upgraded 9 gems by rebuilding Gemfile.lock file.

### CODE REVIEW

**Comment by gomayonqui** [comment]
:+1:

**Comment by jasnow** [comment]
new pull request

**Comment by jasnow** [comment]
@orendon - Can you please restart this Travis run? https://travis-ci.org/orendon/contrib-hub/builds/94512356

**Comment by jasnow** [comment]
Travis is green now so Travis restart is not needed.

File: db/migrate/20120930040540_create_users.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def change
       t.string :name
       t.string :token
 
-      t.timestamps
+      t.timestamps null: false
```
Comment (orendon) [inline db/migrate/20120930040540_create_users.rb]: @jasnow Rails is pretty good at handling `timestamps` for us, any specific reason to change this old migrations? (isn't even reflected on `schema.rb`)

**Pull request merged**